packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -13,6 +13,7 @@ import {
 	type IFluidHandle,
 	type IFluidLoadable,
 } from "@fluidframework/core-interfaces";
+// eslint-disable-next-line import/no-deprecated
 import { type IProvideFluidHandle } from "@fluidframework/core-interfaces/internal";
```

Comment:
@microsoft/fluid-cr-devtools Is there a better change here since the type is deprecated?

Reply:
Maybe, the deprecation node Craig wrote points to something... but we need to think carefully if we'll have to keep something like this for back-compat to identify FF 1.x FluidHandles. I think for now go with the lint rule